fix: "Status" column values in "Custom Status" page table not translated (#34857)
RocketChat/Rocket.Chat: .changeset/seven-otters-worry.md

---
'@rocket.chat/meteor': patch
---

> Fixes an issue where the language preference was not being applied to the custom user status page, specifically for the status options in the table. The fix ensures the status options are translated correctly.

Reviewer: ```suggestion
Fixes an issue where custom status' values in the custom status page table were not being properly translated
```
